src/ResourceManager/ContainerInstance/Commands.ContainerInstance/Commands/NewAzureContainerGroupCommand.cs

```diff
@@ -27,14 +27,10 @@ namespace Microsoft.Azure.Commands.ContainerInstance
     /// <summary>
```

Comment:
Please add a snippet about this change to your changelog to ensure it is picked up in the next release: https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/ContainerInstance/ChangeLog.md